lib/stdlib/test/lists_SUITE.erl

```diff
@@ -2743,6 +2743,20 @@ hof(Config) when is_list(Config) ->
 
     ok.
 
+%% Test lists:enumerate/1 and lists:enumerate/2
+enumerate(Config) when is_list(Config) ->
+    [] = lists:enumerate([]),
+    [] = lists:enumerate(10, []),
+    [{1,a},{2,b},{3,c}] = lists:enumerate([a,b,c]),
+    [{10,a},{11,b},{12,c}] = lists:enumerate(10, [a,b,c]),
+    {'EXIT', {function_clause, _}} = (catch lists:enumerate(0)),
+    {'EXIT', {function_clause, _}} = (catch lists:enumerate(0, 10)),
+    {'EXIT', {function_clause, _}} = (catch lists:enumerate(1.0, [])),
+    {'EXIT', {function_clause, _}} = (catch lists:enumerate(1.0, [a,b,c])),
+    {'EXIT', {function_clause, _}} = (catch lists:enumerate(<<1>>, []),
+    {'EXIT', {function_clause, _}} = (catch lists:enumerate(<<1>>, [a,b,c]),
+    {'EXIT', {function_clause, _}} = (catch lists:enumerate(1, <<1,2,3>>)).
```

Comment:
```suggestion
    {'EXIT', {function_clause, _}} = catch lists:enumerate(0),
    {'EXIT', {function_clause, _}} = catch lists:enumerate(0, 10),
    {'EXIT', {function_clause, _}} = catch lists:enumerate(1.0, []),
    {'EXIT', {function_clause, _}} = catch lists:enumerate(1.0, [a,b,c]),
    {'EXIT', {function_clause, _}} = catch lists:enumerate(<<1>>, []),
    {'EXIT', {function_clause, _}} = catch lists:enumerate(<<1>>, [a,b,c]),
    {'EXIT', {function_clause, _}} = catch lists:enumerate(1, <<1,2,3>>).
```
You forgot some ending parenthesis on line 2756 and 2757. `catch` should no longer need the parenthesis, so I suggest you just remove them.